### ISSUE TITLE
Possibility to reset the switchbot switch status in HA?

### ISSUE THREAD

**Issue by IrealiTY** [devWaves/SwitchBot-MQTT-BLE-ESP32#17]
Basically, because the switchbot doesn’t have an ON and OFF state the reporting in the dashboard will sometimes do weird things and not trigger correctly.
For those who don’t know, when the switchbot triggers, it’ll press down a button/switch/whatever, and then go back to its original position.
This is fine if you use it on a Lightswitch or similar devices that do have a realistic on and off-state by themselves.
For use cases though where the button is always in a neutral position (for example Touch-sensitive buttons like on a coffee machine etc) the status will keep changing even though the device that’s being controlled by the switchbot doesn’t really match that status.

Direct example to make it a bit more clear:
I use the switchbot on an Intercom device to open the door to our apartment complex. As long as the button is pressed, the entrance door will be unlocked. If I call the automation for it, It will press it, wait for a few seconds (that’s a setting on the switchbot device itself) and then let go.
Now Home Assistant will report the status as “ON” and next time I need to use the intercom automation, I’ll need a to use it twice back to back to turn the status back to “OFF” and then to activate it again.

TL;DR:
Is it possible to make switches go “OFF” (without triggering any actions) automatically after they had been turned on?

**Comment by devWaves** [comment]
if I understand correctly, the current version already does that

the esp32 will send state = "OFF" when the bot is in press mode

in HA when using MQTT discovery, the switchbot will go "ON" for split second and will go back to "OFF" as soon as the esp32 gets the mqtt message

you should not need to change the device manually back to off, it does that automatically when the bot is in press mode
